ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,1038 @@
+include::../shared-doc/attributes.adoc[]
+
+= ejb-txn-remote-call: Demonstrates remote EJB calls and transaction propagation
+:author: Ondra Chaloupka
+:level: Intermediate
+:technologies: EJB, JTA, Clustering
+
+[abstract]
+The `ejb-txn-remote-call` quickstart demonstrates remote EJB calls over two application servers of {productName}.
+
+:standalone-server-type: ha
+:archiveType: war
+:uses-h2:
+:requires-multiple-servers:
+:jbds-not-supported:
+
+== What is it?
+
+The `ejb-txn-remote-call` quickstart demonstrates the remote EJB calls over two application servers of {productNameFull}.
+
+== Description
+
+The EJB remote call contains a JTA transaction that is propagated over the EJB call.
+Further, the quickstart demonstrates the transaction recovery, which is run for both servers
+when a failure occurs.
+
+This quickstart contains two maven projects.
+The first maven project represents the caller side, and is intended to be deployed on the first server (`server1`).
+The second project represents the called side. This project is intended to be deployed
+to the other two servers (`server2` and `server2`). The two projects should not be deployed to the same server.
```

Comment:
"to the other two servers (`server2` and `server2`)"

"server2" repeated. I think the second instance is supposed to be "server3".